tests/test_2qbits_gates.py

```diff
@@ -63,7 +63,6 @@ def test_fidelity_and_performance_cnot():
 
     assert analyzer_postprocessed_cnot.performance > analyzer_heralded_cnot.performance > analyzer_klm_cnot.performance
 
-
```

Comment:
With PEP8, there must be two lines between methods outside of Class, can you add back a line please ?